registry.json

```diff
@@ -517,6 +517,21 @@
       "operatorName":"Andre Halvorsen",
       "github":"ADATHOR"
    },
+   {
+      "ticker":"LOKI",
+      "poolId":"pool17j94qfcxvr0xvuz5xl2c8nnt8pcv4v2d753vfxx3p4l4sukhtq0",
```

Comment:
Please can you change this to your pool id, e.g. f758bd7d92dec6dc5a7f23ad20bf53d4dcf76c1ed84c0449289cfc8b